packages/react-native-renderer/src/ReactFabricRenderer.js

```diff
@@ -35,6 +35,10 @@ import UIManager from 'UIManager';
 // This means that they never overlap.
 let nextReactTag = 2;
 
+type HostContext = {
+  isInAParentText: boolean,
+};
```

Comment:
In React Native, we've started adopting a common practice to use:

```
type HostContext = $ReadOnly<{|
  isInAParentText: boolean,
|}>;
```

This makes it so you cannot write to `isInAParentText` and disallows access of properties other than the ones supplied. But I'd rather you use what is common in the React repository.

Reply:
Good point! I've been trying to use exact types when I think of it. I dig the $ReactOnly suggestion too.